app/pages/notifications/moderation.cjsx

```diff
@@ -39,9 +39,12 @@ module.exports = React.createClass
         @setState {reports}
 
   render: ->
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}/"
     notification = @props.notification
     [owner, name] = notification.project_slug.split('/') if notification.project_slug
-    path = if notification.project_id then "/projects/#{owner}/#{name}/talk/moderations" else '/talk/moderations'
+    path = if notification.project_id then "/projects/#{notification.project_slug}/talk/moderations" else '/talk/moderations'
```

Comment:
If you want to make this change, then you can remove line 46: `[owner, name] = notification.project_slug.split('/') if notification.project_slug` as it won't be needed.